### PR TITLE
Fix #3557 Merge profile chooser fragment into 2 fargments

### LINKED ISSUE #3557: Merge profile_chooser_fragment.xml into single xml file
Currently there are 4 versions of `profile_chooser_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

It can happen that it becomes really hard to merge all 4 files into single file, in such cases prefer code-readability over single file. Therefore it can be fine if in the end the code still contains 2 or 3 files.

**Note**: **In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.**

### PR DESCRIPTION
**Pull request by priyatanu**
## Explanation

Merged 2 files of profile_chooser_fragment.xml into one profile_chooser_fragment.xml in layout/profile_chooser_fragment.xml
But for the larger screen size I found that sw600dp profile_chooser_fragment is different to mobile screen layout. So I have merged 2 sw600dp modes profile_chooser_fragment into 1 profile_chooser_fragment.xml inside sw600dp folder

## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [ ] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by priyatanu** [comment]
Profile_Chooser1

**Comment by priyatanu** [comment]
ProfileChooser2

**Comment by priyatanu** [comment]
ProfileChooser3

**Comment by priyatanu** [comment]
ProfileChooser4

**Comment by priyatanu** [comment]
PTAL @rishidyno , @ARJUPTA ,

Kindly review .

I made mistakes while rebasing the old PR while trying to open it https://github.com/oppia/oppia-android/pull/4143

So I created a new PR. sorry for the confusion

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -457,6 +457,12 @@
   <dimen name="profile_chooser_fragment_profile_recycler_view_margin_top">36dp</dimen>
   <dimen name="profile_chooser_profile_view_view_margin_top">60dp</dimen>
```
Comment (rishidyno) [inline app/src/main/res/values/dimens.xml:459]: Please remove extra line.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -457,6 +457,12 @@
...
 
+  <dimen name="profile_chooser_fragment_layout_margin_start">32dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_margin_end">32dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_height">128dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_margin_bottom">32dp</dimen>
+
```
Comment (rishidyno) [inline app/src/main/res/values/dimens.xml:464]: Ditto. Remove this extra line.

**Review by rishidyno — CHANGES_REQUESTED** [review]
@priyatanu Please add SS in a table take reference [here](https://github.com/oppia/oppia-android/pull/3644)

**Comment by rishidyno** [comment]
Also, I can see that you are taking SS separately, use emulator SS feature to Take SS. 
image

File: app/src/main/res/layout-land/profile_chooser_fragment.xml [hunk 3]
```diff
@@ -1,121 +0,0 @@
...
-          android:text="@string/profile_chooser_language"
-          android:textColor="@color/profile_chooser_grey_text"
-          android:visibility="gone"
-          app:layout_constraintEnd_toStartOf="@+id/profile_chooser_language_icon"
-          app:layout_constraintTop_toTopOf="parent" />
-
-        <TextView
-          android:id="@+id/profile_select_text"
-          style="@style/Heading1ViewStart"
-          android:layout_marginStart="36dp"
```
Comment (Arjupta) [inline app/src/main/res/layout-land/profile_chooser_fragment.xml:54]: The style attribute here differs from the one in portrait xml (mobile). you forgot to resolve this.

File: app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml [hunk 4]
```diff
@@ -108,7 +108,8 @@
 
       <TextView
         style="@style/Heading1ViewCenter"
-        android:layout_marginBottom="32dp"
+        android:layout_marginBottom="@dimen/profile_chooser_fragment_layout_margin_bottom"
+        android:gravity="center"
         android:minHeight="48dp"
```
Comment (Arjupta) [inline app/src/main/res/layout-sw600dp/profile_chooser_fragment.xml:113]: can you also explain why a new **gravity** attribute has been introduced?

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 5]
```diff
@@ -25,6 +25,14 @@
...
   <dimen name="profile_chooser_profile_view_margin_end_profile_not_added">24dp</dimen>
   <dimen name="profile_chooser_profile_view_margin_top_profile_not_added">24dp</dimen>
+
+  <!-- Profile Chooser Fragment -->
+  <dimen name="profile_chooser_fragment_layout_margin_start">96dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_margin_bottom">28dp</dimen>
+  <dimen name="profile_chooser_fragment_layout_margin_end">96dp</dimen>
+  <dimen name="profile_chooser_fragment_margin_top_profile_already_added">0dp</dimen>
+  <dimen name="profile_chooser_fragment_margin_top_profile_not_added">84dp</dimen>
+
```
Comment (Arjupta) [inline app/src/main/res/values-sw600dp-land/dimens.xml:35]: If the `profile_chooser_fragment_margin_top_profile_already_added` and `profile_chooser_fragment_margin_top_profile_not_added` are already defined in the portrait xml (mobile). Then we do not need to denfine it here again.

**Review by Arjupta — CHANGES_REQUESTED** [review]
@priyatanu PTAL at the requested changes.

**Comment by priyatanu** [comment]
open

**Comment by priyatanu** [comment]
Hi, 
@rishidyno @ARJUPTA ,

I got extremely bsy with work. So the pull request is stale. I have addressed the comments in my new Pull request
- https://github.com/oppia/oppia-android/pull/4282,

Extremely sorry for the inconvenice

**Pull request closed without merging**